content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -109,16 +120,16 @@ deprecated) `no_traverse` remote configuration option. However, it is possible
 to automatically determine which method would be optimal, as long as the sync
 tool can estimate the approximate total size of the remote.
 
-Consider a case where a dataset being synced contains 100 files. Checking if
-each file exists in the remote individually would take 100 API calls. If we know
-that the API returns 1000 files per page, we can say that if the remote contains
-less than `1000 * 100 = 100,000` files, fetching the full remote listing will be
-faster than checking each file individually, since it will take less than 100
-API calls to complete. If the remote contains more files than this `100,000`
-threshold, it will be faster to check our 100 files individually. _(In terms of
-real world performance, there are other considerations that apply, such as
-different API calls taking different amounts of time to complete, and the amount
-of time it takes to run list comparison operations.)_
+Consider a case where a dataset being synchronized contains 100 files. Checking
```

Comment:
This is pretty clear! The only suggestion I have is for readability, splitting up into a few blocks instead of 1 paragarph. Not sure if it would be easy to make a graphic or figure for this?

Reply:
latest revision is split into more sections and reorganized, I'm not really sure what would work here with regard to graphics either